vspace/src/main/webapp/WEB-INF/views/staff/spaces/spacelist.html

```diff
@@ -34,7 +34,7 @@
     color: var(--secondary);
     font-weight: bold;
     font-size: 0.875rem;
-    font-family: 'Quicksand';
+    font-family: 'Roboto', 'Quicksand';

```

Comment:
remove

Reply:
You mean I should remove the `font-family` line?